updatecli/values.yaml

```
@@ -1,6 +1,6 @@
 github:
-  user: "updatebot"
-  email: "updatebot@olblak.com"
+  user: "Jenkins Infra Bot (updatecli)"
+  email: "60776566+jenkins-infra-bot[bot]@users.noreply.github.com"
```

this is an actual user
```suggestion
  email: "60776566+jenkins-infra-bot@users.noreply.github.com"
```